common/logging.go

```diff
@@ -101,26 +100,23 @@ func setFireLevels(level logrus.Level) []logrus.Level {
 func SetupLogging(loggers []string, logOpts map[string]string, tag string, debug bool) error {
 	setupFormatter()
 
-	// Always setup syslog.
-	valuesToValidate := getLogDriverConfig(Syslog, logOpts)
-	err := validateOpts(Syslog, valuesToValidate, syslogOpts)
-	if err != nil {
-		return err
-	}
-
 	// Logrus has a default logger that outputs to os.stderr. Set this
-	// default output to go to ioutil.Discard to not have duplicate logs.
-	logrus.SetOutput(ioutil.Discard)
-	setupSyslog(valuesToValidate, tag, debug)
+	// default output to go to os.Stdout to not have duplicate logs.
+	logrus.SetOutput(os.Stdout)
```

Comment:
The for loop will never be entered if there is no log-driver parameter passed into cilium. I prefer keeping 'default' for all non-supported loggers.